src/rhsm/config.py

```diff
@@ -199,19 +199,21 @@ def set(self, section, name, value):
                 self.add_section(section)
             super(RhsmConfigParser, self).set(section, name, value)
         if section == "logging" and name == "default_log_level":
-            self.is_log_level_valid(value)
+            self.print_invalid_log_level_msg(value)
 
-    def is_log_level_valid(self, value):
-        valid = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOSET']
-        if value not in valid:
-            print("Invalid Log Level: {lvl}, setting to INFO for this run.".format(lvl=value), file=sys.stderr)
+    def print_invalid_log_level_msg(self, value):
+        valid = ['CRITICAL', 'ERROR', 'WARNING', 'INFO', 'DEBUG', 'NOTSET']
+        valid_level = False
+        if value in valid:
+            valid_level = True
```

Comment:
while this works, it can be simplified a bit -- e.g.:
```python
  valid_level = value in valid
  if not valid_evel:
     ...
```

Reply:
changed

Reply:
much simplier